index.mako

```diff
@@ -912,11 +912,21 @@ the commit will be aborted.
 
 ## Confining hooks to run at certain stages
 
-If pre-commit during push has been installed, then all hooks (by default) will
-be run during the `push` stage.  Hooks can however be confined to a stage by
-setting the `stages` property in your `.pre-commit-config.yaml`.  The
-`stages` property is an array and can contain any of `commit`, `push`,
-`prepare-commit-msg` and `commit-msg`.
+Since the `default_stage` top level configuration property of the
+`.pre-commit-config.yaml` file is set to all stages by default, when installing
+hooks using the `-t`/`--hook-type` option (see [pre-commit
+install [options]](#pre-commit-install)), all hooks will be installed by default
+to run at the stage defined through that option. for instance,
+`pre-commit install --hook-type pre-push` will install by default all hooks
+to run at the `push` stage.
+
+Hooks can however be confined to a stage by setting the `stages` property in
+your `.pre-commit-config.yaml`.  The `stages` property is an array and can
+contain any of `commit`, `push`, `prepare-commit-msg` and `commit-msg`.
```

Comment:
also `manual` (not a regression, but we might as well fix it while fixing this!)